src/content/guides/deploying-istio/index.md

```diff
@@ -0,0 +1,283 @@
++++
+title = "How to deploy Istio in your cluster"
+description = "Tutorial on how to deploy Istio in the differnet providers offered by Giant Swarm."
```

Comment:
"in the differnet providers offered by Giant Swarm" - that doesn't sound right (and the spelling for different isn't right). I'd prefer "supported" over "offered". Also, the cloud providers don't play a big role here in my opinion. How about:

    Tutorial on how to deploy Istio on a Giant Swarm Kubernetes cluster